modal/cli/run.py

```diff
@@ -331,49 +331,46 @@ def serve(
 
 
 def shell(
-    func_ref: Optional[str] = typer.Argument(
+    container_or_function: Optional[str] = typer.Argument(
```

Comment:
Nit, but one tricky thing about `container_or_function` is that they're not equivalent — the "container" mode is a `container_id` but while we do _have_ a `function_id` concept, that isn't what you would pass here 🤔

Reply:
Yeah that makes sense! It's only internal though since the metavar in the help is `REF` so maybe fine.